src/main/java/com/exadel/discountwebapp/location/repository/LocationRepository.java

```diff
@@ -13,4 +14,6 @@
     List<Location> findAllByCountry_CountryCode(String countryCode);
 
     List<Location> findAllByCity(String city);
+
+    List<Location> findLocationByIdIn(Collection<Long> ids);
```

Comment:
Standard org.springframework.data.repository.CrudRepository#findAllById can be used instead

Reply:
fixed it